hfo2/src/arch/aarch64.rs

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright 2019 Sanguk Park.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#![allow(non_camel_case_types)]
```

Comment:
I believe we need to remove it in the longer term.  May I ask if you could leave a comment on that?